index.js

```diff
@@ -25,6 +25,7 @@ const extensions = {
   '.coffee': ['coffee-script/register', 'coffee-script'],
   '.coffee.md': ['coffee-script/register', 'coffee-script'],
   '.csv': 'require-csv',
+  '.eg': ['earlgrey/register', 'earlgrey'],
```

Comment:
If not, you'll need to change the second entry to something like this:
https://github.com/tkellen/js-interpret/blob/master/index.js#L15-L19

FWIW, given how new earlgrey is, I'm guessing you don't need this at all. The array of entries is a list of fallbacks for rechoir to use if something fails with any of them. This makes it possible for us to support older versions that may have differing loader mechanisms.